### PR TITLE
RevenuePool upgrade

### PR DESCRIPTION
**Pull request by xiaoch05**

### CODE REVIEW

File: contracts/auction/RevenuePoolV2.sol [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+    function settleToken(address _tokenAddress) public {
+        uint balance = ERC20(_tokenAddress).balanceOf(address(this));
+
+        // to save gas when playing
+        if (balance > 10) {
+            address pointsRewardPool = registry.addressOf(AuctionSettingIds.CONTRACT_POINTS_REWARD_POOL);
+            address contributionIncentivePool = registry.addressOf(AuctionSettingIds.CONTRACT_CONTRIBUTION_INCENTIVE_POOL);
+            address governorPool = registry.addressOf(AuctionSettingIds.CONTRACT_GOVERNOR_POOL);
+            address devPool = registry.addressOf(AuctionSettingIds.CONTRACT_DEV_POOL);
```
Comment (HackFisher) [inline contracts/auction/RevenuePoolV2.sol:75]: Suggest to continue to use CONTRACT_DIVIDENDS_POOL to save & simplify upgration and gas.

File: contracts/auction/AuctionSettingIds.sol [hunk 2]
```diff
@@ -29,5 +29,6 @@ contract AuctionSettingIds is SettingIds {
 
     bytes32 public constant CONTRACT_DEV_POOL = "CONTRACT_DEV_POOL";
 
-
-}
\ No newline at end of file
+    // 0x434f4e54524143545f474f5645524e4f525f504f4f4c00000000000000000000
+    bytes32 public constant CONTRACT_GOVERNOR_POOL = "CONTRACT_GOVERNOR_POOL";
```
Comment (HackFisher) [inline contracts/auction/AuctionSettingIds.sol:33]: Suggest to continue to use CONTRACT_DIVIDENDS_POOL to save & simplify upgration and gas.

**Review by HackFisher — CHANGES_REQUESTED** [review]

File: contracts/auction/RevenuePoolV2.sol (hunk 1 above)
Comment (xiaoch05) [reply contracts/auction/RevenuePoolV2.sol:75]: ok fix it.

**Review by HackFisher — APPROVED** [review]

**Comment by HackFisher** [comment]
@hujw77 please help review this pull request.

**Pull request merged**